components/remote/ui/ir/CardButtons.qml

```diff
@@ -31,124 +32,141 @@ Item {
-        }
+        Row {
+            Layout.alignment: Qt.AlignHCenter
+            spacing: 32
+            Button {
+                visible: obj.isSupported(Remote.F_DIGIT_4)
+                title: "4"
+                mouseArea.onClicked: { obj.channel(4); }
+            }
 
-        Button {
-            visible: obj.isSupported(Remote.F_DIGIT_6)
-            title: "6"
-            mouseArea.onClicked: { obj.channel(6); }
-        }
+            Button {
+                visible: obj.isSupported(Remote.F_DIGIT_5)
+                title: "5"
+                mouseArea.onClicked: { obj.channel(5); }
+            }
 
-        Button {
-            visible: obj.isSupported(Remote.F_DIGIT_7)
-            title: "7"
-            mouseArea.onClicked: { obj.channel(7); }
+            Button {
+                visible: obj.isSupported(Remote.F_DIGIT_6)
+                title: "6"
+                mouseArea.onClicked: { obj.channel(6); }
+            }
         }
 
-        Button {
-            visible: obj.isSupported(Remote.F_DIGIT_8)
-            title: "8"
-            mouseArea.onClicked: { obj.channel(8); }
-        }
+        Row {
+            Layout.alignment: Qt.AlignHCenter
+            spacing: 32
+            Button {
+                visible: obj.isSupported(Remote.F_DIGIT_7)
+                title: "7"
+                mouseArea.onClicked: { obj.channel(7); }
+            }
 
-        Button {
-            visible: obj.isSupported(Remote.F_DIGIT_9)
-            title: "9"
-            mouseArea.onClicked: { obj.channel(9); }
-        }
+            Button {
+                visible: obj.isSupported(Remote.F_DIGIT_8)
+                title: "8"
+                mouseArea.onClicked: { obj.channel(8); }
+            }
 
-        Button {
-            visible: obj.isSupported(Remote.F_DIGIT_SEPARATOR)
-            title: "-"
-            mouseArea.onClicked: { }
+            Button {
+                visible: obj.isSupported(Remote.F_DIGIT_9)
+                title: "9"
+                mouseArea.onClicked: { obj.channel(9); }
+            }
         }
 
-        Button {
-            visible: obj.isSupported(Remote.F_DIGIT_0)
-            title: "0"
-            mouseArea.onClicked: { obj.channel(0); }
-        }
+        Row {
+            Layout.alignment: Qt.AlignHCenter
+            spacing: 32
+            Button {
+                visible: obj.isSupported(Remote.F_DIGIT_SEPARATOR)
+                title: "-"
+                mouseArea.onClicked: { }
+            }
 
-        Button {
-            visible: obj.isSupported(Remote.F_DIGIT_ENTER)
-            title: "Pre-CH"
-            mouseArea.onClicked: { }
+            Button {
+                visible: obj.isSupported(Remote.F_DIGIT_0)
+                title: "0"
+                mouseArea.onClicked: { obj.channel(0); }
+            }
+
+            Button {
+                visible: obj.isSupported(Remote.F_DIGIT_ENTER)
+                title: "E"
```

Comment:
Is this rendered on the button?
If yes, I think this should be translated. @martonborzak please verify

Reply:
I was a bit confused about this feature. It was not used before so i made placeholder.
The same applies for the DIGIT_SEPERATOR feature. There are no corresponding functions in the remote.cpp.

Reply:
This supposed to be the `enter` key on a keypad. I think these should not be translated as they are to mimic the "real" keys. So I would just call it "ENTER".

Reply:
I don't think ENTER will fit the button since there are 3 buttons in a row

Reply:
Or "ENT" this is on my Samsung remote written

Reply:
There are all kind of labels: "E", "ENT", "OK", graphical ones...  For space reasons I'd use "OK" or "E" and still put it in translations.
I guess @martonborzak needs to decide ;-)

Reply:
I vote for "ENT" as I think that is the closest to "ENTER".